docs/install.md

```diff
@@ -19,7 +19,7 @@
 - [Uninstall](#uninstall)
   - [install.sh](#installsh-1)
   - [Homebrew](#homebrew)
-  - [yarn, npm](#yarn-npm-1)
+  - [npm](#npm-1)
```

Comment:
Will this new heading `#npm-1` work? I didn't know that was a thing.

Reply:
I don't see why not? This is the auto-generated one, and because there are two anchors named the same - one gets the `1` appended.

That was already happening before:
* https://coder.com/docs/code-server/latest/install#yarn-npm
* https://coder.com/docs/code-server/latest/install#yarn-npm-1

I didn't manage to figure out how to serve the documentation locally so couldn't really test... Any tips/guide on how to validate it?

Reply:
Ah! I misread the diff and thought you manually added it. Sadly, docs preview doesn't work on forks since GH Actions won't let you share secrets with forks. At least we haven't had any luck.

Looks good to me - disregard! Sorry for the noise!